packages/open-api-page/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "@hashicorp/react-open-api-page",
   "description": "The OpenApiPage component, and related server utilities, are used to render documentation pages for a provided Swagger 2.0 specification.",
   "author": "HashiCorp",
-  "version": "0.0.1",
+  "version": "2.0.0",
```

Comment:
Manually bumped to avoid conflicts with `2.0.0`, which [was somehow already published](https://registry.npmjs.org/@hashicorp/react-open-api-page).